siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -1226,13 +1234,15 @@ def _check_configs(self):
 
         # LLRF Status
         value = (1 << 2) - 1
-        dev = self._llrf
-        if dev.connected:
-            value = _updt_bit(value, 0, 0)
-            okc = dev['ILK:BEAM:TRIP'] == self._llrf_intlk_state
-            okc &= dev['ILK:MAN'] == self._llrf_intlk_state
-            value = _updt_bit(value, 1, not okc)
-        self.run_callbacks('LLRFStatus-Mon', value)
+        for dev in self._llrfs:
+            if dev.connected:
+                value = _updt_bit(value, 0, 0)
+                fim_orbit = dev.fast_interlock_monitor_orbit
+                fim_manual = dev.fast_interlock_monitor_manual
+                okc = fim_orbit == self._llrf_intlk_state
+                okc &= fim_manual == self._llrf_intlk_state
+                value = _updt_bit(value, 1, not okc)
+            self.run_callbacks('LLRFStatus-Mon', value)
```

Comment:
Doesn't this mean that if one of the LLRFs is connected, it would be enough for the error checking to say everything is ok?

Maybe we should add 2 more LEDs here? `Connected{0,1}` and `Configured{0,1}`

Reply:
Yes this would work, but as it requires changes in the PVs meaning/types, can we keep this fix in mind so @anacso17 can implement when she returns next week?